detect_variants/detect_variants.cwl

```diff
@@ -11,25 +11,32 @@ inputs:
         secondaryFiles: [".fai", "^.dict"]
     tumor_bam:
         type: File
-        secondaryFiles: .bai
+        secondaryFiles: [".bai"]
     normal_bam:
         type: File
-        secondaryFiles: .bai
+        secondaryFiles: [".bai"]
     interval_list:
         type: File
     dbsnp_vcf:
         type: File?
-        secondaryFiles: .tbi
+        secondaryFiles: [".tbi"]
     cosmic_vcf:
         type: File?
-        secondaryFiles: .tbi
+        secondaryFiles: [".tbi"]
     strelka_exome_mode:
         type: boolean
+    pindel_chromosome_list:
```

Comment:
Instead of adding a new input to determine which chromosomes we want to run pindel on, could we pull the chromosomes from the interval list?  For exome, we only run on targeted chromosomes.  For WGS, we will be providing an interval list for Mutect to limit variant detection to the autosomal+sex chromosomes of the genome.